data/sheragi/archaeology missions.txt

```diff
@@ -212,19 +212,37 @@ mission "Deep Archaeology 5"
 		conversation
 			`You breathe a sigh of relief as you land safely in the spaceport on <planet>. "Okay, what was that all about?" you ask.`
 			`	"I honestly don't know," says Albert. "I can tell you what I suspect, though. The history books don't actually say when the Deep was first settled. The first documented expedition to the Deep was in the 24th century, and they found several human colonies already established in the region, claiming to date back to the very dawn of space flight. But I think the first settlers were brought there long before that. Brought there on alien ships."`
+			branch "pug"
+				or
+					has "FWC Pug 2: offered"
+					has "FW Pug 3: offered"
 			choice
 				`	"For what purpose?"`
 					goto purpose
 				`	"What aliens? The Quarg?"`
 					goto aliens
+			label pug
+			choice
+				`	"For what purpose?"`
+					goto purpose
+				`	"Wait, are you talking about the Pug?"`
+					goto surprise
 			label purpose
 			`	"I'm not sure," he says. "But I first became interested in the Deep while researching the Alpha Wars. I'm sure you know the history, how humanity was nearly enslaved by psychopathic 'super' humans that we ourselves had created by dabbling in genetic engineering. At that time, the Deep made a sudden, improbable technological leap that allowed them to construct weapons and ships far more powerful than what the Alphas had, and the war against them was won."`
 				goto end
 			label aliens
 			`	"There are popular folk tales in the Deep about 'elves,'" says Albert. "Thin, pale, graceful creatures, with blue blood."`
 			`	John frowns. "Hemocyanic blood. Not the Quarg, then. Their lungs are odd, but their blood uses hemoglobin for oxygen transport, just like ours."`
+				goto end
+			label surprise
+			`	Albert and John look at each other confusedly before Albert says, "Who are the Pug, exactly?"`
+			`	You go on to explain to them what you know about the Pug, their invasion into human space, and what you have heard of them interfering with human affairs in the Deep. You also give a brief description of what they look like, which puts a surprised look on both of their faces.`
+			`	"Thin, pale, and with blue blood? That's precisely the description that we've had of these aliens based on folk tales," replies Albert.`
+			`	"Well then we have good reason to believe they fit into our current knowledge of the timeline here. However, based on your description Captain <last>, they sound far too dangerous to attempt to seek out, especially after what we just fled from."`
```

Comment:
```suggestion
			`	"Well then, we have good reason to believe they fit into our current knowledge of the timeline here. However, based on your description Captain <last>, they sound far too dangerous to attempt to seek out, especially after what we just fled from."`
```
Instincts tell me there should be a comma there. I can look for a full explanation why if you want.